From 1fe0f6bdfdaf1140c683de052ae11a4f36c6aa37 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Tue, 29 May 2018 15:01:43 +0200
Subject: [PATCH] base-files: Add ntp.nic.cz to default ntp servers

IP addresses are used on top of just domains to allow time
synchronization without working DNSSEC.
---
 package/base-files/files/bin/config_generate | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/package/base-files/files/bin/config_generate b/package/base-files/files/bin/config_generate
index edcc7f9..d0d6bcf 100755
--- a/package/base-files/files/bin/config_generate
+++ b/package/base-files/files/bin/config_generate
@@ -234,6 +234,11 @@ generate_static_system() {
 		set system.ntp='timeserver'
 		set system.ntp.enabled='1'
 		set system.ntp.enable_server='0'
+		add_list system.ntp.server='217.31.202.100'
+		add_list system.ntp.server='195.113.144.201'
+		add_list system.ntp.server='195.113.144.238'
+		add_list system.ntp.server='2001:1488:ffff::100'
+		add_list system.ntp.server='ntp.nic.cz'
 		add_list system.ntp.server='0.openwrt.pool.ntp.org'
 		add_list system.ntp.server='1.openwrt.pool.ntp.org'
 		add_list system.ntp.server='2.openwrt.pool.ntp.org'
-- 
2.17.0

